projects/PM_hash_navigation/main.cc

```diff
@@ -95,14 +97,14 @@ main(int argc, char **argv)
     // ----- camera loop ---------------
     unsigned int frame = 0;
 
+    cv::Mat rotatedView, resizedRotatedView, image, resized;
+    cv::Mat outputImage(unwrapRes, CV_8UC3);
     for (frame = 0;; frame++) {
         // Read from camera
-        if (!cam->readFrame(originalImage)) {
-            return EXIT_FAILURE;
-        }
+        cam->readFrameSync(originalImage);
```

Comment:
When ``readFrame`` returns ``false`` that's because there's no frame available. It doesn't mean that there was an error (you get an exception in this case)